CHANGES.rst

```diff
@@ -49,6 +49,8 @@ https://github.com/zopefoundation/Zope/blob/4.x/CHANGES.rst
 - Update ``AccessControl`` to version 6.2 to mitigate a security problem.
   (CVE-2023-41050)
 
+- Added image dimensions to SVG file properties 
+  `#1146 <https://github.com/zopefoundation/Zope/pull/1146>`_.
```

Comment:
This changelog is now in the paragraph for `5.8.4 (2023-09-06)` which was already released. I made https://github.com/zopefoundation/Zope/pull/1158 to move it to the latest "unreleased" paragraph